include/deal.II/grid/grid_generator.h

```diff
@@ -466,7 +479,9 @@ namespace GridGenerator
    * Each entry must be positive. An empty vector is equivalent to one
    * subdivision in each direction.
    *
-   * @param colorize Assign different boundary ids if set to true.
+   * @param colorize
+   * (see @ref GlossColorization "the glossary entry on colorization")
```

Comment:
Maybe this phrase in parentheses should go after the definition of the parameter (you've done this for `hyper_cube_with_cylindrical_hole`)